Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pickable.py

```diff
@@ -194,6 +215,32 @@ def pickable(cls, commits, repository, commits_story=None, excluded=None):
 
         return [commit for commit in commits if str(commit) in filtered_in]
 
+    @classmethod
+    def related_commits_from_notes(cls, commit, run_command_function=None):
+        '''
+        Parse Git notes for patterns like:
+            Picked-from: abc1234
+            Cherry-picked-from: <hash>
```

Comment:
Probably need something to exclude a commit too, right? Something like `Exclude-because:`